guide/voice/v12.md

```diff
@@ -0,0 +1,50 @@
+# Comparison to v12
+
+Discord.js v11 and v12 are released with built-in voice engines for playing audio over Discord. They have been plagued with issues and have had suffered from volatile performance.
+
+`@discordjs/voice` is a new replacement for these engines and is a separate "add-on" that you can install that aims to solve these problems.
+
+## Improved audio playback
+
+The focus of `@discordjs/voice` was to improve audio playback performance. In Discord.js v12, occasionally streams would suffer from "micro tears" in the audio. This would sound like a small crackle between audio packets, or simply a very small gap between audio packets.
```

Comment:
```suggestion
The focus of `@discordjs/voice` was to improve audio playback performance. In Discord.js v12, streams would occasionally suffer from "micro tears" in the audio. This would sound like a small crackle between audio packets, or simply a very small gap between audio packets.
```